scripts/fetchSbomData.ts

```diff
@@ -0,0 +1,62 @@
+import fs from "fs";
+import fetch from "node-fetch";
+
+interface SBOMData {
+  dependencies?: Record<string, unknown>;
+  [key: string]: unknown; // Add more specific fields if known
+}
+
+interface FetchError extends Error {
+  response?: Response;
+}
+
+const fetchSBOMData = async (url: string): Promise<SBOMData> => {
```

Comment:
we've decided to follow this naming convention for abbrevs. in new code

```suggestion
const fetchSbomData = async (url: string): Promise< GitHubSbomApiResponse > => {
```